### PR TITLE
DR-608 Refinement of the snapshot card

### PR DESCRIPTION
**Pull request by myessail**
image
image
image
image

### CODE REVIEW

**Review by thathert — APPROVED** [review]
LGTM

File: src/components/dataset/query/sidebar/QuerySidebarPanel.jsx [hunk 1]
```diff
@@ -2,15 +2,26 @@ import React from 'react';
...
 import Card from '@material-ui/core/Card';
 import Button from '@material-ui/core/Button';
 import Typography from '@material-ui/core/Typography';
 import HighlightOff from '@material-ui/icons/HighlightOff';
 import { applyFilters } from '../../../../actions';
 
+const styles = theme => ({
+  cardPadding: {
+    padding: '10px',
+  },
```
Comment (RoriCremer) [inline src/components/dataset/query/sidebar/QuerySidebarPanel.jsx:16]: nit: should we be using the theme spacing here?

File: src/components/dataset/query/sidebar/QueryViewSidebar.jsx [hunk 2]
```diff
@@ -49,6 +51,7 @@ const styles = theme => ({
     display: 'none',
   },
   drawer: {
+    backgroundColor: '#f2f0f0',
```
Comment (RoriCremer) [inline src/components/dataset/query/sidebar/QueryViewSidebar.jsx]: also theme?

**Review by RoriCremer — APPROVED** [review]
no changes needed for my approval, but might be worth trying to use the theming a bit more in styling

File: src/components/dataset/query/sidebar/QuerySidebarPanel.jsx (hunk 1 above)
Comment (thathert) [reply src/components/dataset/query/sidebar/QuerySidebarPanel.jsx:16]: Good point

**Pull request merged**